bigframes/session/_io/bigquery.py

```diff
@@ -17,17 +17,37 @@
 from __future__ import annotations
 
 import datetime
+import itertools
 import textwrap
 import types
-from typing import Dict, Iterable, Optional, Union
+from typing import Dict, Iterable, Optional, Sequence, Union
 import uuid
 
 import google.cloud.bigquery as bigquery
 
 IO_ORDERING_ID = "bqdf_row_nums"
+MAX_LABELS_COUNT = 64
 TEMP_TABLE_PREFIX = "bqdf{date}_{random_id}"
 
 
+def create_job_configs_labels(
+    job_configs_labels: Optional[Dict[str, str]],
+    api_methods: Sequence[str],
+) -> Dict[str, str]:
+    # If there is no label set
```

Comment:
This comment is just describing _what_ the code does, not _why_. See: go/comments

```suggestion
```